src/app/docs/[...path]/layout.tsx

```diff
@@ -0,0 +1,24 @@
+import { getPageAST } from "lib/db";
+import "lib/db";
```

Comment:
note: this has been added along with the top level to ensure [the db connection gets closed](https://github.com/mongodb/docs-nextjs/pull/1/files#diff-196dc6cfdb581213173439841d9e85ada909cf40face80a3ac1b6739bbd85bf6R79-R81)

open to suggestions for a better cleanup function

Reply:
thats a great idea! except maybe return the data, and finally close the client